src/main/java/decodes/tsdb/TimeSeriesDb.java

```diff
@@ -485,11 +495,26 @@
 public abstract class TimeSeriesDb
     implements HasProperties, DatabaseConnectionOwner
 {
+    private static final org.slf4j.Logger log = LoggerFactory.getLogger(TimeSeriesDb.class);
     public static String module = "tsdb";
 
     /** The application ID of the connected program */
     protected DbKey appId = Constants.undefinedId;
 
+    /**
+     * Thread pool used for running cache reload tasks.
+     */
+    protected static final ScheduledExecutorService cacheExecutor = Executors.newScheduledThreadPool(1, (r) -> {
```

Comment:
There is also a public cacheExecutor in SqlDatabaseIO.  Do these two cacheExecutors work together?

Reply:
We *could* make the executors part of the settings object, but I'd want to think about how that would affect work like #520 and such. My gut says in a good way, DecodesSetttings could just be the application context vs just settings. but again, definitely larger task than what this PR already is.